config/config.go

```diff
@@ -696,33 +696,34 @@ func trimBOM(f []byte) []byte {
 	return bytes.TrimPrefix(f, []byte("\xef\xbb\xbf"))
 }
 
-func LoadConfigFile(config string) ([]byte, error) {
+func LoadConfigFile(config string) ([]byte, bool, error) {
```

Comment:
can we put a comment here that the bool specifies that the config file is remote?

Reply:
Do you want me to name the return parameter or is a comment sufficient?

Reply:
Up to you, either works for me